src/AdminConsole/Pages/Organization/Verify.cshtml

```diff
@@ -6,16 +6,19 @@
     ViewData["Title"] = "Verify your account";
 }
 
-<p>We've sent a confirmation link to the email address you provided. Click on the link in the email to verify your account</p>
+@section Head {
+    <!-- This to refresh the page to check if the user has verified and logged in -->
+    <meta http-equiv="refresh" content="15"/>
+}
 
-<meta http-equiv="refresh" content="10" />
+<p>We've sent a confirmation link to the email address you provided. Click on the link in the email to verify your account</p>
 
 <environment names="Development">
     @{
         var mailPath = System.IO.Path.GetFullPath("mail.md");
     }
-    <br />
+    <br/>
```

Comment:
Nit: `<br>` is a void element, so it doesn't need the `/` -- `<br>` should work. We can establish dominance over whoever decides to read this code by highlighting that we have read the W3C specification on HTML tags.